src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -111,6 +116,10 @@ class CoreWorkerDirectActorTaskSubmitter
     /// an RPC client to the actor. If this is DEAD, then all tasks in the
     /// queue will be marked failed and all other ClientQueue state is ignored.
     rpc::ActorTableData::ActorState state = rpc::ActorTableData::PENDING;
+    /// How many times this actor has restarted so far. Starts at -1 to
+    /// indicate that the actor is not yet created. This is used to drop stale
+    /// messages from the GCS.
+    int64_t num_restarts = -1;
```

Comment:
Will this variable be updated?

Reply:
Yes, it gets updated each time we hear that the actor is restarted. Is that not clear in the comment?

Reply:
Sorry, I didn't find the code to update this variable.

Reply:
Ah I see, it's not currently getting updated, but I'll fix that.